rules/integrations/github/impact_github_member_removed_from_organization.toml

```diff
@@ -0,0 +1,46 @@
+[metadata]
+creation_date = "2023/10/11"
+integration = ["github"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2023/12/14"
+
+[rule]
+author = ["Elastic"]
+description = """
+A member was removed or their invitation to join was removed from a GitHub Organization.
+"""
+from = "now-9m"
+index = ["logs-github.audit-*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Member Removed From GitHub Organization"
+risk_score = 21
+rule_id = "095b6a58-8f88-4b59-827c-ab584ad4e759"
+severity = "low"
+tags = ["Domain: Cloud",
```

Comment:
What are we attempting to convey to the analyst on the other end? If we want to keep this generic, maybe we make it a BBR instead?

Reply:
I guess I thought organizational changes might be worth checking but you're right it's also a normal process and would probably be better as a BBR rule